sources/packages/web/src/components/aest/students/modals/AssessApplicationChangeRequestModal.vue

```diff
@@ -101,15 +101,10 @@ export default defineComponent({
       if (!validationResult.valid) {
         return;
       }
-      resolvePromise(
-        {
-          note: note.value as string,
-          applicationEditStatus: showParameter.value,
-        },
-        {
-          keepModalOpen: true,
-        },
-      );
+      resolvePromise({
```

Comment:
Should we not use await now?

Reply:
This is the method for resolving the Promise and releasing the `await` on the consumer side.
No await should be applied here.

Reply:
Added